### PR TITLE
Update messages.ts

### PR DESCRIPTION
**Pull request by CaraDune**
#### List of Changes
This PR updates the message users receive after requesting the download of their data. 

#### Motivation and Context
This change is required to be compliant with privacy laws and is requested and approved by the Developer's Privacy & Tech Team on behalf of the DPO.

#### How Has This Been Tested?
This code has not been tested: **reviewers please check markdown and tone of voice**

#### Types of changes

- [X] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by thisisjp — APPROVED** [review]
<3

File: SendUserDataDownloadMessageActivity/messages.ts [hunk 1]
```diff
@@ -14,69 +14,41 @@ export const userDataDownloadMessage = (
...
-  - Se hai l’app File installata, ti si aprirà una finestra in cui potrai visualizzare il pacchetto.
-  - Clicca sul documento e inserisci la password per decomprimere il file.
-  
-  Dispositivi Android:
-  - Premi il link qui sopra
-  - Verrai reindirizzato alla finestra del browser, in cui una notifica a fondo pagina ti informa dello scaricamento in corso.
-  - Al termine dello scaricamento, clicca la notifica per aprire il file.
-  - Seleziona l’app con cui aprire il file zip
-  - Clicca sul documento e inserisci la password per decomprimere il file.  
+  Per una migliore esperienza utente, ti consigliamo di copiare il link che hai ricevuto e aprirlo da PC. Puoi farlo anche da smartphone, ma in tal caso avrai bisogno di un'applicazione apposita per aprire il file in formato .zip.<br>
```
Comment (BurnedMarshal) [inline SendUserDataDownloadMessageActivity/messages.ts:44]:
```suggestion
  Per una migliore esperienza utente, ti consigliamo di copiare il link che hai ricevuto e aprirlo da PC. Puoi farlo anche da smartphone, ma in tal caso avrai bisogno di un'applicazione apposita per aprire il file in formato .zip.
```

File: SendUserDataDownloadMessageActivity/messages.ts [hunk 2]
```diff
@@ -14,69 +14,41 @@ export const userDataDownloadMessage = (
...
-  
-  Puoi richiedere di esercitare il tuo diritto di limitazione e opposizione del trattamento e a chiedere la rettifica del tuo indirizzo email. Potrai inoltre rivolgerti al Garante per la protezione dei dati personali.
-  Con riferimento ai dati dei Servizi offerti dagli Enti Erogatori, tutti i diritti dovranno essere esercitati presso di loro.
-  
-  Puoi trovare tutte le informazioni sul trattamento dei tuoi dati nella nostra [Informativa Privacy](ioit://PROFILE_PRIVACY) che è accessibile in ogni momento nella sezione Profilo/Privacy e Condizioni d’uso.
-  Se ti servono dettagli o informazioni su questi dati, ti invitiamo a scrivere all’indirizzo email dpo@pagopa.it.
+  Ti ricordiamo che puoi sempre esercitare i diritti previsti dagli artt. 15 - 22 del GDPR con riferimento ai dati presenti su IO. Con riferimento ai dati dei servizi offerti dagli enti erogatori, tutti i diritti dovranno essere esercitati presso di loro.
+ 
+  Puoi trovare tutte le informazioni sul trattamento dei tuoi dati nella nostra[Informativa Privacy](ioit://PROFILE_PRIVACY), sempre disponibile in app e all'indirizzo <nibile in app e all’inidirizzo https://io.italia.it/app-content/tos_privacy.html>.
```
Comment (BurnedMarshal) [inline SendUserDataDownloadMessageActivity/messages.ts:50]:
```suggestion
  Puoi trovare tutte le informazioni sul trattamento dei tuoi dati nella nostra[Informativa Privacy](ioit://PROFILE_PRIVACY), sempre disponibile in app e all'indirizzo <https://io.italia.it/app-content/tos_privacy.html>.
```

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]
Some typo

**Review by CaraDune — COMMENTED** [review]
thanks!

**Review by BurnedMarshal — APPROVED** [review]

**Pull request merged**